packages/editor/templates/Crypto Pizzas/README.md

```diff
@@ -42,9 +42,9 @@ The `createRandomPizza` function is the public function called in JavaScript tha
 
 The `generateRandomDna` introduces another new function modifier, [`pure`](https://solidity.readthedocs.io/en/v0.5.12/contracts.html#pure-functions). Pure functions promise not to read from or modify the state, instead they generally return values to another function that does.
 
-The `getPizzasByOwner` function is another public function called by JavaScript to return all pizzas created by the owner of a specified address. The function introduces another modifier, [`view`](https://solidity.readthedocs.io/en/v0.5.12/contracts.html#view-functions) which promise not to modify the state.
+The `getPizzasByOwner` function is another public function called by JavaScript to return all pizzas created by the owner of a specified address. The function introduces another modifier, [`view`](https://solidity.readthedocs.io/en/v0.5.12/contracts.html#view-functions) which promise not to modify the state. The function uses the [`memory`](https://solidity.readthedocs.io/en/latest/introduction-to-smart-contracts.html#storage-memory-and-the-stack) data storage area to only keep a valuable for the life of the contract instance.
```

Comment:
I don't understand this sentence:
> The function uses the [`memory`](https://solidity.readthedocs.io/en/latest/introduction-to-smart-contracts.html#storage-memory-and-the-stack) data storage area to only keep a valuable for the life of the contract instance.

I assume you mean "value" vs. "valuable"? And could you please clarify what this means (for my understanding)? The life of a contract instance? Doesn't a contract instance last forever on the blockchain? Or is this contract instance a JavaScript object?